connectors/flink/src/test/java/io/delta/flink/source/DeltaSourceITBase.java

```diff
@@ -324,7 +324,7 @@ private void assertRowValues(int i, RowData row) {
                     equalTo(Timestamp.valueOf("2022-06-14 18:54:24.547557")
                         .toLocalDateTime().toInstant(ZoneOffset.UTC))
                 );
-                assertThat(row.getString(8).toString(), equalTo(String.valueOf(i)));
+                assertThat(row.getString(8).toString(), equalTo("test-" + i));
```

Comment:
Could you run this test without your fix for SchemaConverter? Just to be sure/check if this will actually fail without that fix. (unless you already did it).

Reply:
when I not changed to include longer strings, this test is pass.

Reply:
Sorry,
I'm not sure if I got it right.

If for a moment, you will revert change from `SchemaConverter and run this test here, that now reads and expects `test-` + i data, it will fail or pass?

Reply:
yes. when I revert change from SchemaConverter and run this test here, that now reads and expects "test-" + i data, it will fail.
it is pass with my change in SchemaConverter